pkg/kubectl/cmd/create_pdb.go

```diff
@@ -60,7 +60,8 @@ func NewCmdCreatePodDisruptionBudget(f cmdutil.Factory, cmdOut io.Writer) *cobra
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.PodDisruptionBudgetV1GeneratorName)
-	cmd.Flags().String("min-available", "1", i18n.T("The minimum number or percentage of available pods this budget requires."))
+	cmd.Flags().String("min-available", "", i18n.T("The minimum number or percentage of available pods this budget requires."))
```

Comment:
Oh! Good point. @kargakis, can you expand on how exactly adding a new generator would fix the backward compatibility? Do we switch on the presence of the `max-unavailable` field?